src/commands/load.ts

```diff
@@ -51,4 +54,27 @@ export class LoadCommand extends BaseCommand {
             }
         }
     }
+
+    private async loadYoutubeList(
+        message: Message, 
+        guild: GuildManager, 
+        youtubeListUrl: string,
+        collection?: string,
+    ): Promise<void> {
+        const result = await getInfoWithArg(youtubeListUrl, ['--flat-playlist']);
+        message.reply("Start loading from youtube playlist, please be patient...");
+        if (Array.isArray(result)) {
+            for (const song of result) {
+                this.logger.info(`Now loading song: ${song.url}`);
+                if (!song.url) continue;
+                const info = await getInfo(song.url);
+                await guild.datasource.saveToPlaylist(
+                    BilibiliSong.withInfo(info, message.author), message.author, collection
+                );
```

Comment:
was parallel execution problematic?

Reply:
Want to keep the order of original playlist

Reply:
The database does not order it anyway, cuz we don't have save timestamp yet